QRCoderTests/ArtQRCodeRendererTests.cs

```diff
@@ -96,7 +96,7 @@ public void can_render_artqrcode_from_helper()
             var bmp = ArtQRCodeHelper.GetQRCode("A", 10, Color.Black, Color.White, Color.Transparent, QRCodeGenerator.ECCLevel.L);
 
             var result = HelperFunctions.BitmapToHash(bmp);
-            result.ShouldBe("57ecaa9bdeadcdcbeac8a19d734907ff");
+            result.ShouldBe("6c1bb8257349b29f76e50c3fa8abd9c9");
```

Comment:
Do you want to change the default size from 0.8 to 1.0 on the helper as well?  Then this hash would have remained the same.

Reply:
See ArtQRCode.cs line 286

Reply:
I've overseen it. Was just on my way fixing it, when I saw your comment.